scripts/spec-check.py

```
@@ -99,8 +115,11 @@ def main():
             errs.append(f"AC id 重复: {ac.get('id')}")
         ids.add(ac.get("id"))
 
-    # 3. blastRadius 元素非空
-    if not all(isinstance(x, str) and x.strip() for x in fm["blastRadius"]):
+    # 3. blastRadius 元素非空（字符串或 {repo,path} 对象皆可）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)